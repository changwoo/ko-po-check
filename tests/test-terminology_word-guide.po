# -*-text-*-
# test 5
#
msgid ""
msgstr ""
"Project-Id-Version: test 5\n"
"POT-Creation-Date: 2002-07-25 01:30+0900\n"
"PO-Revision-Date: 2007-04-17 02:17+0900\n"
"Last-Translator: Changwoo Ryu <cwryu@debian.org>\n"
"Language-Team: Korean <ko@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"

#: test.c:5
msgid "Some Properties"
msgstr "어쩌구 등록 정보"

#: test.c:5
msgid "Properties : attribute"
msgstr "속성: 속성"

#: test.c:5
msgid "Dummy Preferences"
msgstr "어쩌구 설정 사항"

#: test.c:5
msgid "Dummy2 Preferences"
msgstr "저쩌구 환경설정"

#: test.c:5
msgid "Dummy3 Preferences"
msgstr "어쩌구 기본 설정"

#: test.c:5
msgid "About Dummy..."
msgstr "어쩌구에 대하여..."

#: test.c:5
msgid "About Dummy2..."
msgstr "어쩌구 정보..."

#: test.c:5
msgid "_About 어쩌구"
msgstr "어쩌구에 대하여(_A)"

#: test.c:5
msgid "Window font"
msgstr "창 폰트"

#: test.c:5
msgid "window font"
msgstr "창 글꼴"

# E: _ 포함
#: test.c:5
msgid "Dummy Pref_erences"
msgstr "더미 선택 사항(_E)"

# E: & 포함
#: test.c:5
msgid "Dummy Proper&ties"
msgstr "어쩌구 속성(&T)"

# E: 단추
#: test.c:5
msgid "button"
msgstr "버튼"

# E: 삭제
#: test.c:5
msgid "_Delete A"
msgstr "A 지우기(_D)"

# E: 찾기
#: test.c:5
msgid "_Find A"
msgstr "A 검색(_F)"

# E: 검색
#: test.c:5
msgid "_Search A"
msgstr "A 찾기(_S)"

# E: 무른모
#: test.c:5
msgid "This id free open source software"
msgstr "이것은 자유 오픈소스 무른모라고 번역하지 않는다"

#: test.c:5
msgid "Educational software"
msgstr "교육용 소프트웨어라고 번역한다"

# E: 아이템
msgid "%d item(s)"
msgstr "%d개의 아이템이라고 번역하지 않는다"

msgid "We found %d item(s)"
msgstr "%d개의 항목을 잦았다라고 번역한다"

# E: 인스톨, 깔기
msgid "Insert operating system installation media"
msgstr "운영체제 인스톨 미디어를 넣으라고 번역하지 않는다"

msgid "Failed to install theme"
msgstr "테마 깔기에 실패했다고 번역하지 않는다"

msgid "Thisusually means that your installation is corrupted"
msgstr "이는 보통 설치가 깨졌음을 의미한다라고 번역한다"
