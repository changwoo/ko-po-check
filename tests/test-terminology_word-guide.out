-:17: 등록 정보: "properties"은(는) "속성"(이)라고 번역하길 제안합니다
-:25: 설정 사항: "preferences"은(는) "기본 설정"(이)라고 번역하길 제안합니다
-:29: 환경설정: "preferences"은(는) "기본 설정"(이)라고 번역하길 제안합니다
-:37: 대하여: "about"은(는) "정보"(이)라고 번역하길 제안합니다
-:45: 대하여: "about"은(는) "정보"(이)라고 번역하길 제안합니다
-:49: 폰트: "font"은(는) "글꼴"(이)라고 번역하길 제안합니다
-:58: 선택 사항: "preferences"은(는) "기본 설정"(이)라고 번역하길 제안합니다
-:68: 버튼: "button"은(는) "단추"(이)라고 번역하길 제안합니다
-:73: 지우기: "delete"은(는) "삭제"(이)라고 번역하길 제안합니다
-:78: 검색: "find"은(는) "찾기"(이)라고 번역하길 제안합니다
-:83: 찾기: "search"은(는) "검색"(이)라고 번역하길 제안합니다
-:88: 무른모: "software"은(는) "소프트웨어"(이)라고 번역하길 제안합니다
-:96: 아이템: "item"은(는) "항목"(이)라고 번역하길 제안합니다
-:103: 인스톨: "install"은(는) "설치"(이)라고 번역하길 제안합니다
-:106: 깔기: "install"은(는) "설치"(이)라고 번역하길 제안합니다
